### PR TITLE
added toasts to user register form for field validation

### LINKED ISSUE #464: Add Validation
image

- Need all fields to be filled before pressing submit

### PR DESCRIPTION
**Pull request by aditithanekar**

### CODE REVIEW

File: src/components/Forms/Register.jsx [hunk 1]
```diff
@@ -35,6 +36,22 @@ const Register = () => {
   };
 
   const handleSubmit = () => {
+    const incompleteFields = Object.values(user).some(
+      (value) => value === "" || !value
+    );
+    const invalidRequirements = Object.values(requirements).some(
+      (check) => !check.state
+    );
+    const atLeastOneDiet = Object.values(dietaryRestrictions).some(
```
Comment (shahdivyank) [inline src/components/Forms/Register.jsx]: this field is not required, since u could have no diet restrictions

**Comment by shahdivyank** [comment]
Assign urself

File: src/components/Forms/Register.jsx (hunk 1 above)
Comment (aditithanekar) [reply src/components/Forms/Register.jsx]: but we have a checkbox field called None for diet restrictions

**Comment by aditithanekar** [comment]
image

**Review by shahdivyank — APPROVED** [review]

**Review by rfairooz — APPROVED** [review]

**Pull request merged**